spec/classes/foreman_proxy__proxydhcp__spec.rb

```diff
@@ -194,6 +194,37 @@
             'interfaces' => ['eth0']
         ) end
       end
+
+      context "as manager of ACLs for dhcp" do
+        let :facts do
+          facts.merge({:ipaddress_eth0 => '192.168.100.21',
+                       :ipaddress      => '192.168.100.21',
+                       :netmask_eth0   => '255.255.255.0',
+                       :network_eth0   => '192.168.100.0',
+                       :osfamily       => 'RedHat'})
```

Comment:
I'd prefer a conditional test. If RH it should be present, if not RH then it should not be present.

Reply:
Now that the if is gone, you can just leave out all these facts.